tests/unit_tests/io/test_trajectory_loader.py

```diff
@@ -361,6 +364,99 @@ def write_viswalk_csv_file(
     data.to_csv(file, sep=";", index=False, mode="a", encoding="utf-8-sig")
 
 
+def write_vadere_csv_file(
+    *,
+    data: Optional[pd.DataFrame] = None,
+    file: pathlib.Path,
+    frame_rate: float = 0,
+):
+    data = data.rename(
+        columns={
+            ID_COL: "pedestrianId",
+            FRAME_COL: "simTime",
+            X_COL: "startX-PID1",  # "-PID1" stands for processor id 1 and is used in Vadere
+            # outputs as extension of the generic column name startX
+            Y_COL: "startY-PID1",  # "-PID1" see comment above
+        }
+    )
+    data["simTime"] = data["simTime"] / frame_rate
```

Comment:
Wouldn't this be nan for the default `frame_rate` (=0). If so, is the default=0 really needed?